CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.9.0] - 2015-12-21
+### Added
+- Add package.json for npm support
```

Comment:
yea I guess its two different use cases.

`import "wds";` would import the scss file which you then compile. There's also the use case of like `require('wds')` using webpack, which could be the CSS. We'd potentially have a different npm module for the Sass files, I've seen this done before. 

For when we're publishing this to npm, I don't think you should have to build it to get the CSS, which is what federalist is doing now. You should npm install it, and the CSS should be compiled somehwere, and then I think there's even something like a "main-css" directive you can put in package.json to point to it.

Also, what is the plan to publish this on npm?